inc/saischeduler.h

```diff
@@ -71,7 +71,7 @@ typedef enum _sai_scheduler_attr_t
     /**
      * @brief Scheduling algorithm weight
      *
-     * Range [1 - 100].
+     * Range [1 - 255].
```

Comment:
Please update the comments reflecting the 8 bit value range  0..255, where 0 is invalid value

Reply:
@JaiOCP Updated the range to 0...255 and 0 as invalid weight